### DISCUSSION TITLE
Als Camera FOV

### DISCUSSION THREAD

**Discussion by Snakeyio** [Sixze/ALS-Refactored#177] [Q&A]
Any idea how I could access Als Camera field of view and modify it in real time not settings, ex ZOOM FOV in when aiming etc.

Can I do it via blueprints? there's no set FOV variable..

**Comment by ameaninglessname** [comment]
This may involve several steps:

1. extend the als camera animation blueprint, to provide "FOV override curve"(bOverrideFov) and "FOV value curve"(the FOV wanted)
2. blend these values(fov) in `UAlsCameraComponent::TickCamera`

Not sure if this is the good way to do it. Besides I think the camera animation blueprint may also need to be layered if the camera logic goes bigger.

**Reply by Snakeyio** [reply]
how do I extend the als camera animation blueprint to provide FOV override curve?
